### PR TITLE
adding compat data for HTML global attributes

### PR DESCRIPTION
**Pull request by frittate**
https://github.com/mdn/browser-compat-data/issues/914

### CODE REVIEW

**Comment by Elchi3** [comment]
Hey, thanks for your pull request! 💯 

If you want to validate the JSON locally, you can also do so using `npm test`. See also our [contributing file](https://github.com/mdn/browser-compat-data/blob/master/CONTRIBUTING.md#validating-the-data). If you have any more questions, feel free to ask them here as well.

**Comment by frittate** [comment]
OK, got it. I get the same error: `global_attributes should NOT have additional attributes.`
How should the file be structures instead?

File: html/global_attributes.json [hunk 1]
```diff
@@ -0,0 +1,886 @@
...
+        }
+      },
+      "contextmenu": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Global_attributes/class",
+          "support": {
+            "chrome": {
+              "version_added": true,
+              "version_removed": "52"
+            },
```
Comment (jpmedley) [inline html/global_attributes.json]: `chrome_android` and `webview_android` should match this.

File: html/global_attributes.json [hunk 2]
```diff
@@ -0,0 +1,886 @@
...
+      },
+      "spellcheck": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Global_attributes/spellcheck",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": "64",
```
Comment (jpmedley) [inline html/global_attributes.json]: This is 47, not 64. 

https://www.chromestatus.com/features/5653714173952000

File: html/global_attributes.json [hunk 3]
```diff
@@ -0,0 +1,886 @@
...
+            "deprecated": false
+          }
+        }
+      },
+      "spellcheck": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Global_attributes/spellcheck",
+          "support": {
+            "chrome": {
+              "version_added": true
```
Comment (jpmedley) [inline html/global_attributes.json]: This should be 9.

https://www.chromestatus.com/features/5653714173952000

File: html/global_attributes.json [hunk 4]
```diff
@@ -0,0 +1,886 @@
...
+            },
+            "safari": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": "9.3",
+              "partial_implementation": false
+            },
+            "webview_android": {
+              "version_added": "4.4",
```
Comment (jpmedley) [inline html/global_attributes.json]: This should be 47. Webview version numbers follow Chromium, not Android.

File: html/global_attributes.json [hunk 5]
```diff
@@ -0,0 +1,888 @@
...
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": true
+          }
+        }
+      },
+      "data-*": {
```
Comment (Elchi3) [inline html/global_attributes.json]: This is not a valid identifier unfortunately. (the error message is not helpful, so you couldn't know this.
`"data-*": {`

Better would be:
```
"data_attributes": {
  "__compat": {
    "description": "<code>data-*</code> attributes",
    "mdn_url": "...",
    "support": {
```

**Pull request merged**